plio/utils/covariance.py

```diff
@@ -3,8 +3,8 @@
 
 def compute_covariance(lat, lon, rad, latsigma=10., lonsigma=10., radsigma=15., semimajor_axis=None):
     """
-    Given geospatial coordinates, desired accuracy sigmas, and an equitorial radius, computes the rectangular
-    covariance matrix using error propagation.
+    Given geospatial coordinates, desired ground distance accuracies in meters (sigmas), and an equatorial 
```

Comment:
"uncertainties" is a better word than "accuracies" here. Generally the precision of a measurement is captured by uncertainties, not the accuracy. Accuracy is a more complex beast that often requires comparison with some outside data.

Reply:
![actually](https://c.tenor.com/tv3Yr2A2I4kAAAAd/nerd-ackchyually.gif)

Reply:
I like it! The wording, not the GIF. I also changed it to read "ground **location** uncertainties" as opposed to distances